UKHomeOffice/cerberus-service
# COP 9617: Render risk indicators for rule matches

## Description
- Added new `Table` component
- Render risk indicators table at the bottom of each rule match, if they exist. The title of this field should be `Risk indicators (X)`
- If not, only render `Risk indicators (0)`
- Created a very simple test for this
- Updated a fixture for the test

## To Test (Before process-models!527 is merged)
- Run the UI, click on a task and scroll down to the `Rules matched` section. By default, each should have a `Risk indicators (0)` title.
- Add a line after TaskVersions:119, updating `firstRule.childSets` with some input data and toggle `firstRule.hasChildSets` to `true`. If done correctly, this should render the table with the correct data.

## To Test (After process-models!527 is merged)
- Run the UI, click on a task and the risk indicators' data should be displayed in the `Risk indicators (X)` field.

## Developer Checklist

\* Required
